pandas_gbq/gbq.py

```diff
@@ -956,7 +956,10 @@ def _wait_for_job(job):
             bytes_billed = int(query_job._properties["statistics"]["query"].get("totalBytesProcessed", 0))
             bytes_processed = int(query_job._properties["statistics"]["query"].get("totalBytesBilled", 0))
             print("Total bytes billed (processed): %s (%s)" % (sizeof_fmt(bytes_billed),sizeof_fmt(bytes_processed)))
-    query_results = query_job.results()
+    try:
+        query_results = query_job.results()
+    except:
+        query_results = query_job.result()
```

Comment:
If we require 0.26 or later, we could always use `.result()` and remove `_wait_for_job` logic from here.